File: AutoserviceWPF1/AutoserviceWPF/Models/ModelsDB/Order.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace AutoserviceWPF.Models.ModelsDB
{
    public partial class Order
    {
        public Guid Id { get; set; }

        public DateTime Date { get; set; }

        public string Time { get; set; }

        public string Comment { get; set; }

        [JsonProperty("Client_Id")]
        public Guid ClientId { get; set; }

        [JsonProperty("Employee_Id")]
        public Guid? EmployeeId { get; set; }

        [JsonProperty("Status_Id")]
        public int StatusId { get; set; }

        public virtual Client Client { get; set; }

        public virtual Employee Employee { get; set; }

        [JsonProperty("List_Consumable_Parts")]
        public virtual ICollection<ListConsumablePart> ListConsumableParts { get; set; } = new List<ListConsumablePart>();

        [JsonProperty("List_Services")]
        public virtual ICollection<ListService> ListServices { get; set; } = new List<ListService>();

        public virtual StatusOrder Status { get; set; }
    }
}

File: Application/Client/car-service/src/interface/model/order.ts
import { UUID } from "crypto";
import ServiceList from "./list_service";
import ConsumablePartsList from "./list_consumable_parts";
import Status from "./status";
import Employee from "./employee";

export default interface Order{
    id?:UUID,
    date?:string,
    time?:string,
    comment?:string,
    client_id?:UUID,
    employee_id?:UUID,
    status_id?:number,
    list_services?:ServiceList[]
    list_consumable_parts?:ConsumablePartsList[]
    status?:Status,
    employee?:Employee
}